### PR TITLE
feat(repo): Added remaining reth metrics

### PR DESCRIPTION
**Pull request by rwlockbg**

### CODE REVIEW

File: bin/btc-server/src/pegout_scheduler.rs [hunk 1]
```diff
@@ -601,6 +604,10 @@ impl PegoutScheduler {
         // some of the blocks we already saw might not be in the node's chain.
         // To avoid errors related to this, we'll just ask called to wait.
         if is_syncing(bitcoind)? {
+            if let Some(telemetry) = telemetry {
```
Comment (arminsabouri) [inline bin/btc-server/src/pegout_scheduler.rs]: Nit: this exact code is duplicated across a bunch of files. can we extract this to a macro somehow?
LogTelemetry!(self.telemetry, SyncError::NodeNotSynced.to_string())

**Review by arminsabouri — COMMENTED** [review]
The reth / authority metrics. I'll take a deeper look tomorrow but just wanted to leave a stylistic comment

File: bin/btc-server/src/pegout_scheduler.rs (hunk 1 above)
Comment (rwlockbg) [reply bin/btc-server/src/pegout_scheduler.rs]: Done, fixed!

**Review by scottmillner — APPROVED** [review]
🔥 🔥 🔥

File: .codespellrc [hunk 2]
```diff
@@ -1,3 +1,3 @@
 [codespell]
 skip = .git,target,./crates/storage/libmdbx-rs/mdbx-sys/libmdbx,Cargo.toml,Cargo.lock,pnpm-lock.yaml,./contracts/lib/,btc_server.bin,./testing/ef-tests/ethereum-tests/
-ignore-words-list = crate,ser,ratatui,ot,tE
+ignore-words-list = crate,ser,ratatui,ot,tE,commet
```
Comment (arminsabouri) [inline .codespellrc:3]: It should be spelled comet not commet. So i think this change can be reverted. 
Nit

**Review by arminsabouri — APPROVED** [review]
Awesome stuff

**Pull request merged**